new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDomain.java

```diff
@@ -214,156 +224,281 @@ public void testCreateMiiDomain() {
+            "welcome3", domainNamespace1),
+            String.format("createSecret failed for %s", adminSecretName));
+
+    // create encryption secret
+    logger.info("Create encryption secret");
+    String encryptionSecretName = "encryptionsecretdomain3";
+    assertDoesNotThrow(() -> createDomainSecret(encryptionSecretName, "weblogicencdomain3",
+            "weblogicencdomain3", domainNamespace1),
+             String.format("createSecret failed for %s", encryptionSecretName));
+
+    // create the domain CR
+    int adminNodePort = 30714;
+    createDomainResource(domainUID, domainNamespace1, adminSecretName, repoSecretName,
+            encryptionSecretName, adminNodePort, replicaCount);
+
+    // wait for the domain to exist
+    logger.info("Check for domain custom resouce in namespace {0}", domainNamespace1);
+    withStandardRetryPolicy
+        .conditionEvaluationListener(
+            condition -> logger.info("Waiting for domain {0} to be created in namespace {1} "
+                   + "(elapsed time {2}ms, remaining time {3}ms)",
+                domainUID,
+                domainNamespace1,
+                condition.getElapsedTimeInMS(),
+                condition.getRemainingTimeInMS()))
+        .until(domainExists(domainUID, DOMAIN_VERSION, domainNamespace1));
+
+
+    // check admin server pod exist
+    logger.info("Check for admin server pod {0} existence in namespace {1}",
+            adminServerPodName, domainNamespace1);
+    checkPodCreated(adminServerPodName, domainUID, domainNamespace1);
+
+    // check managed server pods exists
+    for (int i = 1; i <= replicaCount; i++) {
+      logger.info("Check for managed server pod {0} existence in namespace {1}",
+              managedServerPrefix + i, domainNamespace1);
+      checkPodCreated(managedServerPrefix + i, domainUID, domainNamespace1);
+    }
+
+    // check admin server pod is running
+    logger.info("Wait for admin server pod {0} to be ready in namespace {1}",
+            adminServerPodName, domainNamespace1);
+    checkPodRunning(adminServerPodName, domainUID, domainNamespace1);
+
+    // check managed server pods are running
+    for (int i = 1; i <= replicaCount; i++) {
+      logger.info("Wait for managed server pod {0} to be ready in namespace {1}",
+              managedServerPrefix + i, domainNamespace1);
+      checkPodRunning(managedServerPrefix + i, domainUID, domainNamespace1);
+    }
+
+    logger.info("Check admin service {0} is created in namespace {1}",
+            adminServerPodName, domainNamespace1);
+    checkServiceCreated(adminServerPodName, domainNamespace1);
+
+    // check managed server services created
+    for (int i = 1; i <= replicaCount; i++) {
+      logger.info("Check managed server service {0} is created in namespace {1}",
+              managedServerPrefix + i, domainNamespace1);
+      checkServiceCreated(managedServerPrefix + i, domainNamespace1);
+    }
+  }
+
+
   public void tearDown() {
```

Comment:
The @AfterEach annotation is removed from tearDown() method, and the method is only called in this class. Change it to private.

Reply:
done